README.md

```diff
@@ -158,3 +162,15 @@ test('something', async () => {
   expect(context.doSomething).toHaveBeenCalledTimes(3)
 })
 ```
+
+### Parallel Processing + Ordering
+
+By default, the abstractions in `@lifeomic/delta` (`DynamoStreamHandler` and `SQSMessageHandler`) will process events in parallel. To control the parallelization, specify a `concurrency` value when creating the handler.
+
+These abstractions also guarantee automatically ensure that within a batch of events correct _ordering_ is maintained according to the ordering semantics of the upstream event source, even when processing in parallel.
```

Comment:
`guarantee automatically ensure`?  Should this be `These abstractions also ensure that ...`

Reply:
great catch. Fixed.